### PR TITLE
SB-23851 - Integrated App APIs - Onboarding

### PR DESCRIPTION
**Pull request by maheshkumargangula**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- N/A

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5799,4 +5800,22 @@ kong_apis:
...
       - name: request-size-limiting
-        config.allowed_payload_size: "{{ small_request_size_limit }}"
\ No newline at end of file
+        config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: integrationAppRead
+    uris: "{{ integration_app_prefix }}/v1/read"
+    upstream_url: "{{ content_service_url }}/app/v4/read"
+    strip_uri: true
+      plugins:
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5809]: indentation issue for the yaml

Comment (maheshkumargangula) [reply ansible/roles/kong-api/defaults/main.yml]: fixed it.

**Comment by SMYALTAMASH** [comment]
Api is as per the sunbird standards and can be merged after it is documented.

Please let me know if the api sheet and documentation is complete for this API.

**Comment by maheshkumargangula** [comment]
@SMYALTAMASH - Below is the API documentation PR. API Details sheet also updated.
https://github.com/project-sunbird/sunbird.org-docs/pull/756

cc: @gandham-santhosh

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**